nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/probes/ProbesObject.java

```diff
@@ -51,7 +54,8 @@
     @Produces(MediaType.APPLICATION_JSON)
```

Comment:
to avoid duplicate the same type of returned type, you can put `@Produces(MediaType.APPLICATION_JSON)` on the class directly instead method per method.

in other hand i think we will rework the way to add the javadoc... please see https://github.com/nuxeo/nuxeo-management-rest-api/pull/16#discussion_r338301693